scripts/generate_files/ebi_search.py

```diff
@@ -0,0 +1,235 @@
+#!/usr/bin/env python3
+
+import sys
+import argparse
+import MySQLdb
+import configparser
+from datetime import datetime
+from lxml import etree as ET
+
+
+def get_g2p_version(db_host: str, db_port: int, db_name: str, user: str, password: str) -> str:
+    """
+    Fetch the latest G2P version from the database.
+
+    Args:
+        db_host (str): G2P database host name
+        db_port (int): G2P database port number
+        db_name (str): G2P database name
+        user (str): user with read access
+        password (str): password
+
+    Returns:
+        str: G2P version
+    """    
+    try:
+        db = MySQLdb.connect(host=db_host, port=db_port, user=user, passwd=password, db=db_name)
+        cursor = db.cursor()
+
+        sql = """   SELECT m.version
+                    FROM meta m
+                    LEFT JOIN source s ON s.id = m.source_id
+                    WHERE s.name = 'G2P'
+                    ORDER BY m.date_update DESC LIMIT 1
+            """
+
+        cursor.execute(sql)
+        g2p_version = cursor.fetchone()
+        cursor.close()
+        db.close()
+
+    except MySQLdb.Error as error:
+        print("Database connection failed:", error)
+        sys.exit(1)
+
+    return g2p_version[0]
+
+
+def dump_g2p_records(db_host: str, db_port: int, db_name: str, user: str, password: str) -> dict[str, dict]:
```

Comment:
Question ? 
Instead of this method, that does a lot of JOIN to the database, can we use the API instead? 
So we use the lgd endpoint for each panel or we can use the download api and then get the details from the record we actually need for the XML. 
Thanks